docs/api-testing/composer.md

```diff
@@ -2,82 +2,601 @@
-5. Make sure you're now in the **Compose** tab (aka Composer) with the [**Input**](/api-testing/composer/#input-sets) and [**Visual**](/api-testing/composer/#visual-view-and-code-view) views toggled.<br/><img src={useBaseUrl('img/api-fortress/2020/09/addComponentView.png')} alt="Add Components from Composer" width="500" />
-  :::note Composing tests
-  You can use either [**Visual** composer](#visual-view-and-code-view) (guides you through building components - no coding required) or [**Code** composer](#visual-view-and-code-view) (requires you to write code from scratch). For this guide, we're using **Visual**.
-  :::
+1. In Sauce Labs, click **API Testing**.
 
-To go back and edit a Test at any time, go into your Project's **Tests** tab, hover over a test name, then click **Edit Test** (pencil icon).<br/><img src={useBaseUrl('img/api-fortress/2020/09/editTest.png')} alt="Components available" width="600" />
+  <img src={useBaseUrl('/img/api-testing/api-testing-nav.png')} alt="Navigating to API Testing" width="400"/>
 
-### Add Test Components
-The next of set steps is to add the necessary test components. When combined, they act as our test logic. You can choose from [I/O (Request) Components](/api-testing/composer/io-components), [Assertion Components](/api-testing/composer/assertion-components/), and [Logical Components](/api-testing/composer/logical-components/).
+2. On the **Projects** page:
+  * If the project has no tests yet, in the **Write your own test** box, click **Use Composer**.
 
-#### Add I/O Request Component
-Create a simple `GET` request and validate that response is correct.
+  <img src={useBaseUrl('/img/api-testing/composer-nav.png')} alt="Navigating to the Composer" width="700"/>
 
-6. Click the **Add Component** button (**+** icon).<br/><img src={useBaseUrl('img/api-fortress/2021/01/addComponent.png')} alt="Add Request Component" width="400"/>
-7. Select the `GET` request component.<br/><img src={useBaseUrl('img/api-fortress/2021/01/getRequest.png')} alt="GET request Component"/>
-8. In the **Url** field, write `https://api.us-west-1.saucelabs.com/rest/v1/public/tunnels/info/versions`. This URL will return a `json` response body.
-9. In the **Variable** field, write `payload`. This variable stores the response.<br/><img src={useBaseUrl('img/api-fortress/2021/01/getFields.png')} alt="GET request fields"/>
-10. Leave the rest of the fields blank and click **Save**. This is what the end result will look like:<br/><img src={useBaseUrl('img/api-fortress/2021/01/getRequestEndResult.png')} alt="GET request end result"/>
+  * If your project has tests, click **Create Test** and then click **From Scratch**.
 
-:::tip
-For detailed information, see [Request (I/O) Components](/api-testing/composer/io-components/).
-:::
+  <img src={useBaseUrl('/img/api-testing/test-create-from-scratch-nav.png')} alt="Navigating to the New Test window" width="450"/>
```

Comment:
@LoryApiFortress If you want to have a discussion offline about screenshot standards and guidelines, please set something up. Otherwise, I ask that you trust me to do my job.